trial-config-service/.github/workflows/pr-test.yml

```diff
@@ -0,0 +1,39 @@
+name: Test Pull Request
```

Comment:
I don't think this works anymore. You should delete this from the file and add this service name to the workflow in the common folder (/.github/workflows/docker...)

Reply:
I have removed this file but added

      - name: Publish Test Report
          if: ${{ always() }}
          uses: actions/upload-artifact@v2
          with:
            name: postman-reports
            path: test-reports/newman-report.html

to docker-build-push